README.md

```diff
@@ -146,10 +146,15 @@ subtyping.  Then
 
 `@CalledMethods(`*set1*`) T1` &#8849; `@CalledMethods(`*set2*`) T2` iff  *set1 &supe; set2* and T1 &#8849; T2.
 
-No `@CalledMethodsPredicate` annotation is ever a subtype of another, or of
-any `@CalledMethods` annotation.  (This imprecise behavior is a limitation
-of the Object Construction Checker.)  For this reason, programmers usually only
-write `@CalledMethodsPredicate` annotations on formal parameters.
+Subtyping between two `@CalledMethodsPredicate` annotations is determined by checking
+whether the proposed subtype implies the proposed supertype. In
+particular:
+
+`CalledMethodsPredicate(`P`) T1` &#8849; `CalledMethodsPredicate(`Q`) T2` iff T1 &#8849; T2
+and &#0172; (P &#8658; Q) is unsatisfiable.
+
+If either P or Q contains operators other than `&&` or `||`, the result of the comparison
```

Comment:
Is this still true?  If the input is invalid, I think it's better to issue an error than to continue.  I suggest changing the behavior (or removing the statement if it is not true).